src/iter/mod.rs

```diff
@@ -581,15 +563,6 @@ pub trait ParallelIterator: Sized {
         find_first_last::find_last(self, predicate)
     }
 
-    #[doc(hidden)]
-    #[deprecated(note = "parallel `find` does not search in order -- use `find_any`, \\
-    `find_first`, or `find_last`")]
-    fn find<P>(self, predicate: P) -> Option<Self::Item>
```

Comment:
Would you mind reverting this part in another PR?

The idea was to help people who port over a sequential iterator and guide them to the right thing.